src/hooks/api/labels.ts

```diff
@@ -1,23 +1,62 @@
-import { useAuthenticatedQuery } from '@/hooks/api/authenticated-query';
+import {
+  PaginationOptions,
+  prefetchAuthenticatedQuery,
+  queryOptions,
+  ServerSideQueryOptions,
+  useAuthenticatedQuery,
+} from '@/hooks/api/authenticated-query';
 import type { Headers } from '@/hooks/api/types/Headers';
 import { Label } from '@/types/Offer';
 import { PaginatedData } from '@/types/PaginatedData';
 import { fetchFromApi, isErrorObject } from '@/utils/fetchFromApi';
 
-const getLabelsByQuery = async ({
+export const labelsToTableData = (labels: Label[]) =>
```

Comment:
The exports are defined at the bottom of the page...
```suggestion
const labelsToTableData = (labels: Label[]) =>
```